internal/store/postgres/migrations/000015_create_jobs_queue_table.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS jobs_queue
+(
+    -- Job specification.
+    id              TEXT      NOT NULL PRIMARY KEY,
+    type            TEXT      NOT NULL,
+    run_at          TIMESTAMP NOT NULL,
+    payload         bytea     NOT NULL,
```

Comment:
what is the consideration choosing `bytea` instead of using other data type (eg: `JSONB`)? since we marshing to json anyway for the payload

Reply:
The payload doesn't _have_ to be a JSON value. In case of delete asset job, the payload is just the URN (string). We are not encoding it as a JSON, just converting `string` to `[]byte`. So `bytea` would be more flexible.